src/App.svelte

```diff
@@ -31,8 +32,16 @@
     connectionDialogState,
   } from './script/stores/connectDialogStore';
   import { isLoading } from 'svelte-i18n';
+  import { fetchBrowserInfo } from './script/utils/api';
+  let isPotentiallyNextGenUser: boolean = false;
 
   onMount(() => {
+    (async () => {
+      const { country } = await fetchBrowserInfo();
```

Comment:
I think we can call this conditionally on whether the user has already seen the redirect dialog.

Reply:
That makes sense, I have added logic for checking before fetching